app/controllers/api/entities/job.rb

```diff
@@ -0,0 +1,18 @@
+module API
+  module Entities
+    class Job < Grape::Entity
+      expose :id
+      expose :name
+      expose :description
+
```

Comment:
Why empty lines? 🤔

Reply:
I thought it's a nicer structure when we separate text fields, flags, dates and references. But I see that it's not very obvious. Will remove it again :D